react/features/toolbox/components/native/Toolbox.js

```diff
@@ -133,7 +133,7 @@ function _mapStateToProps(state: Object): Object {
         _styles: ColorSchemeRegistry.get(state, 'Toolbox'),
         _visible: isToolboxVisible(state),
         _width: state['features/base/responsive-ui'].clientWidth,
-        _reactionsEnabled: getFeatureFlag(state, REACTIONS_ENABLED, false)
+        _reactionsEnabled: getFeatureFlag(state, REACTIONS_ENABLED, true)
```

Comment:
Can you please rebase? This change shouldn't be visible since it was already enabled by default. At any rate, please use the encapsulated function to check.